pyflakes/test/test_other.py

```diff
@@ -989,6 +989,14 @@ def test_returnOnly(self):
         """Do not crash on lone "return"."""
         self.flakes('return 2')
 
+    @skipIf(version_info < (3, 6), 'new in Python 3.6')
+    def test_f_string(self):
+        """Test PEP 498 f-strings are treated as a usage."""
+        self.flakes('''
+        baz = 0
+        print(f'\x7b4*baz\N{RIGHT CURLY BRACKET}')
```

Comment:
why the strange syntax here?

The strange syntax doesn't actually affect the test since it gets evaluated **before** pyflakes evaluates it:

```pycon
>>> '''
...         baz = 0
...         print(f'\x7b4*baz\N{RIGHT CURLY BRACKET}')
... '''
"\n        baz = 0\n        print(f'{4*baz}')\n"
```

And if you wanted to _literaly_ check that, it wouldn't be a valid f-string placeholder:

```pycon
>>> baz = 0
>>> print(f'\x7b4*baz\N{RIGHT CURLY BRACKET}')
{4*baz}
```

This should just be written as `{4*baz)` I imagine?